src/hotspot/share/runtime/objectMonitor.inline.hpp

```diff
@@ -143,6 +143,21 @@ inline void ObjectMonitor::increment_recursions(JavaThread* current) {
   _recursions++;
 }
 
+inline void ObjectMonitor::inc_unmounted_vthreads() {
+  assert(_unmounted_vthreads >= 0, "");
```

Comment:
```suggestion
  assert(_unmounted_vthreads >= 0, "invariant");
```
Here and below - thanks.

Reply:
Done.